src/server/pps/cmds/cmds.go

```diff
@@ -438,6 +439,32 @@ each datum.`,
 		follow      bool
 		tail        int64
 	)
+
+	// prettyLogsPrinter helps to print the logs recieved in different colours
+	prettyLogsPrinter := func(message string) {
+		informationArray := strings.Split(message, " ")
+		if len(informationArray) > 1 {
+			debugString := informationArray[1]
+			debugLevel := strings.ToLower(debugString)
+			var debugLevelColoredString string
+			if debugLevel == "info" {
+				debugLevelColoredString = color.New(color.FgGreen).SprintFunc()(debugString)
+			} else if debugLevel == "warning" {
+				debugLevelColoredString = color.New(color.FgYellow).SprintFunc()(debugString)
+			} else if debugLevel == "error" {
+				debugLevelColoredString = color.New(color.FgRed).SprintFunc()(debugString)
+			} else {
+				debugLevelColoredString = debugString
+			}
+			informationArray[1] = debugLevelColoredString
+			coloredMessage := strings.Join(informationArray[:], " ")
```

Comment:
No need to make a copy of the slice here, i.e. this should work and be slightly faster:

```go
coloredMessage := strings.Join(informationArray, " ")
```